docs/getting-started.md

```diff
@@ -35,7 +35,11 @@ npx create-next-app@latest --typescript
 yarn create next-app --typescript
 ```
 
-After the installation is complete, follow the instructions to start the development server. Try editing `pages/index.js` and see the result on your browser.
+After the installation is complete, follow the instructions to start the development server. Run `npm run dev` or `yarn dev`. This starts the development server on `http://localhost:3000`.
+
+Visit `http://localhost:3000` to view your application.
+
+Now try editing `pages/index.js` and see the result on your browser.
```

Comment:
Can we convert these into steps please 😊 
```suggestion
After the installation is complete:
-  Start the development server by running `npm run dev` or `yarn dev`. This starts the development server on `http://localhost:3000`.
- Visit `http://localhost:3000` to view your application.
- Now try editing `pages/index.js` and see the result on your browser.
```

or something similar

Reply:
Yes of course, that's a nice one. :)